finetune/model.py

```diff
@@ -41,7 +44,26 @@ class PredictMode:
     EXPLAIN = "EXPLAIN"
 
 
-def language_model_op(X, M, params, featurizer_state, mode, encoder):
+def mask_logits(logits, encoder, mask_extra_toks=False):
+    
+    logit_mask = np.zeros(
+    [1, logits.get_shape().as_list()[-1]], dtype=np.float32
+    )
+    logit_mask[:, encoder.vocab_size:] = -np.inf
```

Comment:
What's this extra tail-end of stuff from?

Reply:
That's necessary to zero out the OOV bits.

Reply:
This masks out positional info as it's embedded in the same array.